crates/turborepo-lib/src/config/mod.rs

```diff
@@ -564,7 +558,7 @@ mod test {
 
         repo_root
             .join_component("turbo.json")
-            .create_with_contents(r#"{"experimentalSpaces": {"id": "my-spaces-id"}}"#)
```

Comment:
Can we keep this test so we don't hard-error if users still have this key?

This PR when released would break our repository with:
```
[0 olszewski@macbookpro] /Users/olszewski/code/vercel/turborepo $ turbo_dev build
 WARNING  No locally installed `turbo` found. Using version: 2.4.2.
turbo 2.4.2

turbo_json_parse_error

  × Failed to parse turbo.json.
  ╰─▶   × Found an unknown key `experimentalSpaces`.
           ╭─[turbo.json:221:3]
       220 │   },
       221 │   "experimentalSpaces": {
           ·   ────────────────────
       222 │     "id": "space_WTrAuWxqVUapGgZDX7KiZV1v"
           ╰────
```

Reply:
Do we need to keep this exact test? This is more a config layering test than a parsing test. I can keep the turbo.json parsing tests for the `experimentalSpaces` key